### PR TITLE
[NERLPLANNER] Add remove button to routers

### PR DESCRIPTION
**Pull request by leondavi**
- Add remove button to routers
- Fix exception of frequency

### CODE REVIEW

File: src_py/nerlPlanner/JsonDistributedConfig.py [hunk 1]
```diff
@@ -207,9 +207,14 @@ def add_router(self, router : Router):
             self.main_dict[KEY_ROUTERS][router_name] = router
         return True
     
+    def remove_router(self, router_name : str):
+        self.remove_entity_from_device(router_name)
```
Comment (leondavi) [inline src_py/nerlPlanner/JsonDistributedConfig.py:211]: Typo

File: src_py/nerlPlanner/main.py [hunk 2]
```diff
@@ -119,7 +119,7 @@
                  [sg.Button("Add", size=(9), key=KEY_ROUTERS_BUTTON_ADD, enable_events=True),
                   sg.Button("Load",size=(9), key=KEY_ROUTERS_BUTTON_LOAD, enable_events=True),
                    sg.Button("Save",size=(9), key=KEY_ROUTERS_BUTTON_SAVE, enable_events=True),
-                  sg.Button("Remove",size=(9), key=KEY_CLIENTS_BUTTON_REMOVE, enable_events=True)],
+                  sg.Button("Remove",size=(9), key=KEY_ROUTERS_BUTTON_REMOVE, enable_events=True)],
```
Comment (leondavi) [inline src_py/nerlPlanner/main.py:122]: @NoaShapira8  Why did you change this?

File: src_py/nerlPlanner/Handlers.py [hunk 3]
```diff
@@ -285,6 +285,12 @@ def routers_handler(window,event,values):
             routers_this_router_policy = routers_this_router.get_policy().get_policy_name()
             routers_update_inputs_ui(window)
 
+    if event == KEY_ROUTERS_BUTTON_REMOVE:
+        routers_this_router_name = values[KEY_ENTITIES_ROUTERS_LISTBOX][0] if values[KEY_ENTITIES_ROUTERS_LISTBOX] else None  # protects from bypassing load with selection from KEY_DEVICES_SELECTED_ENTITY_COMBO
```
Comment (ohad123) [inline src_py/nerlPlanner/Handlers.py:289]: fix that

**Review by ohad123 — APPROVED** [review]
